.github/workflows/build.yml

```diff
@@ -66,3 +66,20 @@ jobs:
         run: |
           echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
           bash hack/docker_build.sh ${{ steps.extract_branch.outputs.branch }}
+
```

Comment:
better to move e2e configs to a separate yaml

Reply:
I was trying to integrated build and test in one workflow, So we can launch e2e against every pull. dependents relation can't be set between workflow(separated yaml).